.github/workflows/request-reviews.yml

```diff
@@ -0,0 +1,24 @@
+name: Request reviews for PRs
+
+on:
+  pull_request_target:
+    types: [opened]
```

Comment:
We need to include only:

```
on:
  push:
    paths:
      - deps/**
 ```

to filter `deps/` changes

Reply:
This review requester is also for other file, so checking paths here wouldn't work.

Reply:
Which file?

Reply:
*files, I meant it's checking reviews for all files based on the code owners.